main.go

```diff
@@ -36,6 +37,38 @@ func ping(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+/* Handles a call to /books/(exected id here) and redirects depending on the requested action.  */
+func bookById(w http.ResponseWriter, r *http.Request) {
+	method := r.Method
+	if method == http.MethodGet {
+		getBookById(w, r)
+		return
+	} else {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		return
+	}
+}
+
+/* Return the book with that specific ID. */
+func getBookById(w http.ResponseWriter, r *http.Request) {
+	//Isolating ID:
+	justId, _ := strings.CutPrefix(r.URL.Path, "/books/")
+	id, err := uuid.Parse(justId)
+	if err != nil {
+		log.Println(err)
+		responseJSON(w, http.StatusBadRequest, errIdInvalidFormat)
+		return
+	}
+	//Searching for that ID on database:
+	empty, returnedBook := searchById(id)
+	if empty {
+		w.WriteHeader(http.StatusNoContent) //WE SHOULD DECIDE HOW TO ANSWER THIS. MAYBE A 404(NOTFOUND), BUT THAT WOULD BE A CLIENT ERROR, WHAT I THINK IS NOT THE CASE.
```

Comment:
If the user sends a valid uuid, but we don't have any book with that uuid its a `404`.
If the user sends a invalid uuid, you are catching it in the previous error check.
If we have an unknown issue while trying to find the book in the data base, its a `500`.